sdk/multiclients/modular-multiclients/package.json

```diff
@@ -0,0 +1,177 @@
+{
+  "name": "@msinternal/multiclient",
+  "version": "1.0.0-beta.1",
+  "description": "\"Multi Client Service\"",
+  "sdk-type": "client",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "react-native": {
+    "./dist/index.js": "./dist-esm/src/index.js"
+  },
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./types/src/index.d.ts",
+      "require": "./dist/index.cjs",
+      "import": "./dist-esm/src/index.js"
+    },
+    "./api": {
+      "types": "./types/src/api/index.d.ts",
+      "require": "./dist/api/index.cjs",
+      "import": "./dist-esm/src/api/index.js"
+    },
+    "./rest": {
+      "types": "./types/src/rest/index.d.ts",
+      "require": "./dist/rest/index.cjs",
+      "import": "./dist-esm/src/rest/index.js"
+    },
+    "./bar": {
+      "types": "./types/src/bar/index.d.ts",
+      "import": "./dist-esm/src/bar/index.js"
+    },
+    "./foo": {
+      "types": "./types/src/foo/index.d.ts",
+      "import": "./dist-esm/src/foo/index.js"
+    },
+    "./bar/api": {
+      "types": "./types/src/bar/api/index.d.ts",
+      "import": "./dist-esm/src/bar/api/index.js"
+    },
+    "./bar/rest": {
+      "types": "./types/src/bar/rest/index.d.ts",
+      "import": "./dist-esm/src/bar/rest/index.js"
+    },
+    "./foo/api": {
+      "types": "./types/src/foo/api/index.d.ts",
+      "import": "./dist-esm/src/foo/api/index.js"
+    },
+    "./foo/rest": {
+      "types": "./types/src/foo/rest/index.d.ts",
+      "import": "./dist-esm/src/foo/rest/index.js"
+    }
+  },
```

Comment:
One concern here is that if we have N sub clients, and there will be 3*(N + 1) sub path to export here.

Reply:
Giving that multi-api has the same structure as multi-client per this design https://github.com/Azure/azure-sdk-for-js/blob/c369610dd5161a9a22827a39fb1eb30f4eebe121/design/multi-version-design.md, I can't image a case where it's multi-api plus multi-client :(